linux/elements.bsh

```diff
@@ -616,6 +616,32 @@ else
   }
 fi
 
+if [ "${BASH_VERSINFO[0]}${BASH_VERSINFO[1]}" -ge "44" ]; then
```

Comment:
why not use requirements:meet_requirements

Reply:
Habit and this is simple and faster. meet_requirements is very universal and always works, but this I know the second digit doesn't exceed 10. It's a pretty common way to test bash version.